views/newsletters.html

```diff
@@ -295,7 +295,7 @@
     'schedule': __('Schedule'),
 
     'allSendingPausedHeader': __('All sending is currently paused!'),
-    'allSendingPausedBody': __('Your key to send with MailPoet is invalid.'),
+    'allSendingPausedBody': __('Your [link]API key[/link] to send with MailPoet is invalid.'),
```

Comment:
@pavel-mailpoet Do you create new strings or is it ok to modify the exiting one?

Reply:
It is fine to modify the existing one. However, it is discouraged to do if you can. Obviously in this case you cannot avoid changing the translation string. All of the translations will be lost and the translators will have to translate them again. That will take some time and users will see this note in English for some time.